bodhi/server/consumers/masher.py

```diff
@@ -999,57 +1112,65 @@ def _get_master_repomd_url(self, arch):
         return master_repomd % (self.release.version, arch)
 
 
-class MashThread(threading.Thread):
-    """
-    A Thread that performs the subprocess call to mash.
-
-    Attributes:
-        log (logging.Logger): A logger for the thread to use.
-        mash_cmd (basestring): The command to run, including arguments.
-        name (basestring): The tag being mashed. This is set so that the thread name will appear in
-            the logs.
-        success (bool): True if the subprocess finished with exit code 0, False if the process is
-            still running or exited with a non-0 exit code.
-        tag (basestring): The tag being mashed.
-    """
+class RPMMasherThread(MasherThread):
+    ctype = ContentType.rpm
+    pungi_template_config_key = 'pungiconf.rpm'
+
+    def copy_additional_pungi_files(self, pungi_conf_dir, template_env):
+        self.git_clone('https://pagure.io/fedora-comps.git',
+                       'master',
+                       os.path.join(pungi_conf_dir, 'comps'))
```

Comment:
Dusty had suggested using Pungi to do this - I'm slightly inclined to make that a later commit/release just for reducing churn, but if you think that's easy we can do it now.

Reply:
yeah that's what I was suggesting with https://github.com/fedora-infra/bodhi/pull/1900#issuecomment-336984945 - this is a large enough change that I would say we should just go ahead and leverage the new functionality in pungi

Reply:
That's pretty easy.
When I looked at the documentation for it, I couldn't find a way to make it run "make", which in fedora-comps is required.
But after looking at the code, you can do that, so this entire call nd the "make" call can just be dropped.